apps/modernization-ui/src/components/ButtonActionMenu/ButtonActionMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from '@trussworks/react-uswds';
+import { ReactElement, useState, useRef, useEffect } from 'react';
+import styles from './buttonActionMenu.module.scss';
+import { Icon as NBSIcon } from 'components/Icon/Icon';
+
+type Props = {
+    label: string;
+    items: ReactElement[];
```

Comment:
suggestion:  Instead of passing elements the items could be an object that describes the actions.  That way the buttons would also be controlled by this component.  The type would need a label and a function to call when picked.